app/src/renderer/connectors/lcdClientMock.js

```diff
@@ -244,16 +234,21 @@ module.exports = {
     state.keys.push(key)
     return { name, password, seed, address: key.address }
   },
-  async listKeys() {
-    return state.keys.map(k => ({
-      name: k.name,
-      address: k.address
-    }))
-  },
-  async getKey(name) {
-    return state.keys.find(k => k.name === name)
+
+  delete: async (account, { name, password }) => {
+    let key = state.keys.find(k => k.name === name)
+    if (key.password !== password) {
+      throw new Error(`Passwords do not match`)
+    }
+    state.keys = state.keys.filter(k => k.name !== name)
   },
-  async updateKey(account, { name, old_password, new_password }) {
+
+  get: async name =>
+    name === `seed`
```

Comment:
I have the feeling that putting the request for a seed here will cause confusion. This is correctly mirroring the API but a developer wouldn't know of this `magic` key name. I would add another method to query for a seed. But not feeling too strongly.

Reply:
this is handling the case when it calls `GET keys/{name}` where `{name} = seed`. I also think we should change it but on upstream (_i.e_ rename the endpoint on the LCD)

Reply:
I agree with both of you:  We should fix it upstream and then create a new API to mirror the fixed LCD when that happens.

Reply:
I don't think the local API needs to reflect the actual API 100%. The same way the actual API doesn't need to (more like shouldn't) reflect the DB or state machine imo. I think we should fix this here and adjust the endpoints when the upstream change is implemented.

Reply:
OK, I'll do it in this one.